test/immutable-docs-spec.js

```diff
@@ -1,90 +1,262 @@
 var testHelper = require('../etc/test-helper.js');
 
-describe('Immutable document validation parameter', function() {
+describe('Immutable document validation:', function() {
   beforeEach(function() {
     testHelper.init('build/sync-functions/test-immutable-docs-sync-function.js');
   });
 
-  it('can create a document if the old document does not exist', function() {
```

Comment:
It will help to view the changes in this file with `?w=1` to ignore whitespace changes.